mongodb_migrations/config.py

```diff
@@ -5,8 +5,8 @@
 
 
 class Execution(Enum):
-    DOWNGRADE = 'execution_downgrade'
-    MIGRATE = 'execution_migrate'
+    DOWNGRADE = 'downgrade'
```

Comment:
You should have kept the namespace in the string for an enum, It might conflict with other enums in the system